github_actions/spec/dependabot/github_actions/file_parser_spec.rb

```diff
@@ -428,5 +475,6 @@ def mock_service_pack_request(nwo)
         end
       end
     end
+
```

Comment:
💅 Looks like this stray newline found its way into the diff. Feel free to ignore 🙂 

```suggestion
```

Reply:
Nice catch, it sounds like RuboCop also complained. Fixing!